Backend/app/Http/Controllers/KebabController.php

```diff
@@ -0,0 +1,690 @@
+        ]);
+
+        try {
+            $kebab = Kebab::findOrFail($kebabId);
+
+            $comments = $kebab->comments ?? [];
+
+            $newComment = [
+                "id_user" => $request->input("id_user"),
+                "comment" => $request->input("comment"),
+            ];
+
+            $comments[] = $newComment; 
+
+            $kebab->comments = $comments;
+
+            $kebab->save();
+
+            return response()->json([
+                "message" => "Comment added successfully.",
+                "comments" => $kebab->comments,
+            ], 200);
+        } catch (Exception $e) {
+            return response()->json([
+                "error" => "Could not add comment.",
+                "message" => $e->getMessage(),
+            ], 500);
+        }
+    }
+
+    public function removeComment(Request $request, $kebabId)
+    {
+        $request->validate([
+            "id_user" => "required|integer", 
+            "comment" => "required|string",
+        ]);
+
+        try {
+            $kebab = Kebab::findOrFail($kebabId);
+
+            $comments = $kebab->comments ?? [];
+
+            $comments = array_filter($comments, fn($item) => $item["id_user"] !== $request->input("id_user") || $item["comment"] !== $request->input("comment"));
+
+            $kebab->comments = array_values($comments); 
+
+            $kebab->save();
+
+            return response()->json([
+                "message" => "Comment removed successfully.",
+                "comments" => $kebab->comments,
+            ], 200);
+        } catch (Exception $e) {
+            return response()->json([
+                "error" => "Could not remove comment.",
+                "message" => $e->getMessage(),
+            ], 500);
+        }
+    }
+
+    public function updateGoogleReviews(Request $request, $kebabId)
+    {
+        $request->validate([
+            "google_reviews" => "nullable|numeric|between:0,99.9",
+        ]);
+
+        try {
+            $kebab = Kebab::findOrFail($kebabId);
+
+            $googleReviews = $request->input("google_reviews") !== null ? $request->input("google_reviews") : null;
+
+            $kebab->google_reviews = $googleReviews;
+
+            $kebab->save();
+
+            return response()->json([
+                "message" => "Google reviews updated successfully.",
+                "google_reviews" => $kebab->google_reviews,
+            ], 200);
+        } catch (Exception $e) {
+            return response()->json([
+                "error" => "Could not update Google reviews.",
+                "message" => $e->getMessage(),
+            ], 500);
+        }
+    }
+
+    public function updatePyszneplReviews(Request $request, $kebabId)
+    {
+        $kebab = Kebab::findOrFail($kebabId);
+        $kebab->pysznepl_reviews = $request->input("pysznepl_reviews");
+        $kebab->save();
+
+        return response()->json(["message" => "Pyszne.pl reviews updated successfully."]);
+    }
```

Comment:
If I got it right, we should do that using a google and pyszne.pl API or something like that. Definitelly not using local database